developer-guide/concepts-and-designs/multicluster/architecture.md

```diff
@@ -0,0 +1,54 @@
+# Multiple Cluster Architecture
+
+## Introduction
+
+Today, it's very common for organizations to run a handful of Kubernetes Clusters. Each kubernetes cluster is a relatively self-contained unit. And there are several teams working on tying all clusters, such as kubefed. More and more KubeSphere users and customers have expressed their kneed interests in managing all clusters through KubeSphere.
```

Comment:
kneed interests? Is kneed a good word here?

Reply:
typo, kneen